modules/graphics/mrblib/font.rb

```diff
@@ -0,0 +1,17 @@
+module Moon
+  class Font
+    class << self
+      attr_accessor :default_shader
+    end
+
+    alias :initialize_wo_shader :initialize
```

Comment:
Hence the "in the future" part. If you do restructuring as I suggest later, we don't need aliases at all.